bin/anthology/venues.py

```diff
@@ -67,59 +72,101 @@ def dump(self, directory):
         with open(f"{directory}/yaml/venues.yaml", "wt") as f:
             print(yaml.dump(self.venue_dict, allow_unicode=True), file=f)
 
+    @staticmethod
```

Comment:
This method is twice in the code -- as it is static, it should be easy to remove one copy.